File: heroes-app/src/app/models/entity-type-dto.ts
export class EntityTypeDto {
    public id: number;
    public name: string;
    public isBadGuy: boolean;
    public latitude: number;
    public longitude: number;

    constructor(id, isBadGuy, name) {
        this.id = id;
        this.isBadGuy = isBadGuy;
        this.name = name;
        this.latitude = 0;
        this.longitude = 0;
    }
}

File: Fabrit.Heroes/Fabrit.Heroes.Data.Business/Hero/EntityTypeDto.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Fabrit.Heroes.Data.Business.Hero
{
    public class EntityTypeDto
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public bool IsBadGuy { get; set; }
        public double Latitude { get; set; }
        public double Longitude { get; set; }
    }
}
